arch/arm/boot/dts/bcm270x.dtsi

```diff
@@ -130,6 +130,14 @@
 			/* Add alias */
 			status = "disabled";
 		};
+
+		axiperf: axiperf {
+			compatible = "brcm,bcm2835-axiperf";
+			reg = <0x7e009800 0x100>,
+			      <0x7ee08000 0x100>;
+			firmware = <&firmware>;
+			status = "okay";
```

Comment:
@pelwell Are you happy for this to be enabled by default even though the driver won't be built?

Reply:
I think I'd rather have it disabled at this level then enabled higher up, e.g. by bcm2708-rpi.dtsi (even though the two are equivalent).

In general it seems to be frowned on to introduce nodes with an explicit status of "okay", since that is the default.

Reply:
I have no idea what to put here, I copied that section from another section, so do not know what the status parameter actually means.